### ISSUE TITLE
test_reduce fails for size = 3

### ISSUE THREAD

**Issue by magoorden** [UPPAALModelChecker/UCDD#19]
The tests in this library are not runned frequently. For #16, I am creating a test case and thus enabling running `test_cdd` again. I noticed that the current test fails for `test_reduce` with input `size = 3`:

```
test_cdd: /home/martijn/Documents/UCDD/test/test_cdd.cpp:223: void test_reduce(size_t): Assertion `cdd2 == cdd3' failed.
```

**Comment by mikucionisaau** [comment]
Yes, that's the reason the test with size=10 is disabled for 64bits.
Interestingly it passes for 32 bits.
Which means there is some aweful C code messing with integer/pointer sizes.
